packages/core/src/components/table/table.tsx

```diff
@@ -28,11 +28,8 @@ export class Table implements ComponentInterface {
   @Prop({ reflect: true })
   noModal = false;
 
-  /**
-   * Indicates whether the table is currently horizontally scrollable.
-   */
-  @Prop({ reflect: true })
-  isResponsive = false;
+  @State()
+  isResponsive?: boolean;
```

Comment:
Hier maakten we misbruik van de `@Prop()` decorator. Deze property is nooit onderdeel van de public API van dit component geweest, maar met `{ reflect: true }` konden we 'm zetten en stijlen.

Dit vervangen door de property een internal state property te maken en de `<Host>` functional component te gebruiken om attributen op de host te zetten.

Daarnaast is het type veranderd: `is-responsive` wordt gebruikt om te bepalen of het component klaar is.